doc/_admin-guide/110_Template_and_rewrite/000_Customize_message_format/004_Macros_of_syslog-ng.md

```diff
@@ -458,7 +458,7 @@ following macros are available in {{ site.product.short_name }} version 3.9 and
 
 ## ${UNIQID}
 
-A globally unique ID generated from the HOSTID and the
+A globally unique ID generated from the `HOSTID` and the
 RCPTID in the format of HOSTID@RCPTID. For details, see
```

Comment:
`HOSTID` and RCPTID have an and in between them, it should be surrounded with ``-s